src/components/checkbox/checkbox.html

```diff
@@ -0,0 +1,22 @@
+<div class="md-checkbox-layout">
+  <div class="md-checkbox-inner-container">
+    <div class="md-checkbox-frame"></div>
+    <div class="md-checkbox-background">
+      <svg version="1.1"
+           class="md-checkbox-checkmark"
+           xmlns="http://www.w3.org/2000/svg"
+           xmlns:xlink="http://www.w3.org/1999/xlink"
+           viewBox="0 0 24 24"
+           xml:space="preserve">
```

Comment:
I don't think `xmlns:xlink` is but I'd argue `xml:space` is because it explicitly instructs browsers how to parse the SVG XML. From the [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xml:space)

> Note that this attribute influences the way a browser should parse the xml content and therefore will change the way the DOM is built.